### PR TITLE
Add devextreme-internal-tools package to package.json

### PR DESCRIPTION
**Pull request by viterobk**

### CODE REVIEW

**Review by IlyaKhD — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -114,7 +115,9 @@
     "build-themes": "gulp style-compiler-themes",
     "build-themebuilder-assets": "gulp style-compiler-tb-assets",
     "dev": "gulp dev",
-    "test-env": "node testing/launch"
+    "test-env": "node testing/launch",
+    "update-ts": "dx-tools declaration-validator & dx-tools ts-generator",
```
Comment (AlekseyMartynov) [inline package.json]: Single `&` is not cross-platform. It means background in POSIX.

**Review by AlekseyMartynov — CHANGES_REQUESTED** [review]
- Correct single `&`
- Resolve conflicts

**Comment by AlekseyMartynov** [comment]
I think we need to describe the new commands in [README_DEVELOPERS](https://github.com/DevExpress/DevExtreme/blob/19_1/README_DEVELOPERS.md). 

Per @IlyaKhD 's 

> если забыл обновить ТС - Дрон упадет. Матюгнулся, обновил, подлил)

it's becoming a crucial part of the development process.

**Comment by IlyaKhD** [comment]
> it's becoming a crucial part of the development process.

It's just the same thing as "I've forgotten to run unit tests locally and drone fails". Don't forget to run tests before creating a PR, don't forget to update TS when editing declarations - rules of the same kind.

**Comment by AlekseyMartynov** [comment]
Unit tests are well-known among developers. Plus we have the 'Tests and CI' section in `README_DEVELOPERS`.

Automatic generation of TS declarations is our custom approach. It is important that a contributor has to commit a generated file. I think we should briefly describe it by adding a new block to the document.

**Comment by IlyaKhD** [comment]
>I think we should briefly describe it by adding a new block to the document.

Agree :ok_hand:

**Review by AlekseyMartynov — APPROVED** [review]

**Review by BingoRUS — APPROVED** [review]

**Review by IlyaKhD — APPROVED** [review]

**Pull request merged**